examples/chip-tool/commands/discover/Commands.h

```diff
@@ -22,27 +22,26 @@
 #include "DiscoverCommissionablesCommand.h"
 #include "DiscoverCommissionersCommand.h"
 #include <controller/DeviceAddressUpdateDelegate.h>
-#include <lib/mdns/Resolver.h>
+#include <lib/dnssd/Resolver.h>
 
-constexpr uint16_t kMdnsPort = 5353;
+constexpr uint16_t kDnssdPort = 5353;
```

Comment:
Let's leave all 5353 constants as `kMdnsPort` as that's a port used only in the multicast servers.